ADAL/tests/unit/ios/ADKeychainTokenCacheTests.m

```diff
@@ -400,14 +295,12 @@ - (void)testRemoveAllForUserId
 
     ADAssertNoError;
     XCTAssertEqual([self count], 4);
-    XCTAssertEqual([self tombstoneCount], 0);
     
     //remove items with user ID as @"eric@contoso.com" and client ID as TEST_CLIENT_ID
-    [mStore removeAllForUserId:@"eric@contoso.com" error:&error];
+    [mStore wipeAllItemsForUserId:@"eric@contoso.com" error:&error];
```

Comment:
nit: should we test somewhere that the "wipe" entry was created?

Reply:
yes most definately.